contributor-book/src/SUMMARY.md

```diff
@@ -0,0 +1,14 @@
+- [Overview](./overview.md)
+- [How to Read This Book](./how-to-read-this-book.md)
+- [Getting Started](./getting-started/ReadMe.md)
+  - [setting up the environment](./getting-started/setting-up-the-environment.md)
+  - [configuring your editor(optional)](./getting-started/configuring-your-editor.md)
+  - [testing](./getting-started/testing.md)
+- [Architecture Overview](./project-architecture/ReadMe.md)
+  - [Modules](./project-architecture/module.md)
+  - [Serialization](./project-architecture/serialization.md)
+  - [Tensor](./project-architecture/tensor.md)
+  - [Backend](./project-architecture/backend.md)
+- [Guides for Contributors](./guides/ReadMe.md)
+  - [adding a new operation to burn](./guides/adding-a-new-operation-to-burn.md)
```

Comment:
I think we should move the guide from https://github.com/tracel-ai/burn/blob/main/burn-import/DEVELOPMENT.md to the book as well.